### PR TITLE
Added Health Check results

### PR DESCRIPTION
**Pull request by obushnell**
I modified docker.Inspect to return the Health Check information. This solves issue #508.

### CODE REVIEW

File: modules/docker/inspect.go [hunk 1]
```diff
@@ -56,13 +59,44 @@ type VolumeBind struct {
...
 type inspectOutput struct {
 	Id      string
 	Created string
 	Name    string
 	State   struct {
+		Health   struct {
+			Status        string
+			FailingStreak uint8
+			Log           []HealthLog
+		}
```
Comment (brikis98) [inline modules/docker/inspect.go:99]: Could you use the `HealthCheck` struct defined above here?

**Comment by obushnell** [comment]
I changed the struct and added 2 tests. I don't know what the successful health check is for nginx.

**Comment by brikis98** [comment]
The tests failed:

```
goimports................................................................Failed
hookid: goimports

Files were modified by this hook. Additional output:

modules/docker/inspect.go
modules/docker/inspect_test.go
```

Please run `goimports` to reformat the code!

**Comment by obushnell** [comment]
I updated the formatting.

**Comment by brikis98** [comment]
Great, thanks! I just kicked off the tests again.

**Comment by obushnell** [comment]
I made no changes to tests involving aws. I do not believe this failure is related to my changes.

**Comment by brikis98** [comment]
`TestEmptyS3Bucket` failed, which is unrelated to this change, and is known to fail occasionally due to transient issues. I'll proceed to merge this, thank you!

**Comment by brikis98** [comment]
https://github.com/gruntwork-io/terratest/releases/tag/v0.27.1

**Pull request merged**